pkg/apis/application/v1alpha1/app_catalog_types.go

```diff
@@ -1,10 +1,66 @@
 package v1alpha1
 
 import (
+	"encoding/json"
+
 	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const appCatalogCRDValidationSchema = `
+  openAPIV3Schema:
```

Comment:
```suggestion
openAPIV3Schema:
```

Yes the formatting looks strange but two spaces should be OK. Please remove the leading whitespace as that's how we usually format these constants.

Reply:
Fixed formatting - there were tabs, replaced them with two spaces.